docs/querying/api.md

```diff
@@ -433,13 +433,35 @@ URL query parameters:
   series from which to read the label values. Optional.
 - `limit=<number>`: Maximum number of returned series. Optional. 0 means disabled.
 
-
 The `data` section of the JSON response is a list of string label values.
 
-This example queries for all label values for the `job` label:
+This example queries for all label values for the `job_name` label:
+
+```json
+$ curl http://localhost:9090/api/v1/label/job_name/values
+{
+   "status" : "success",
+   "data" : [
+      "node",
+      "prometheus"
+   ]
+}
+```
+
+For label names that contain characters that are not valid in the legacy
+Prometheus character set (letters, numbers, underscores, and colons), they
+should be escaped using the Values Escaping method:
```

Comment:
While it's handy to have a short description here, do we have a comprehensive explanation of this encoding method that we can link to?

Reply:
@beorn7 I can file an issue for the doc issue, is that enough to unblock this PR?

Reply:
Linking to the Prometheus proposal is fine. I.e. link to https://github.com/prometheus/proposals/blob/main/proposals/2023-08-21-utf8.md#text-escaping (as we did for PGW, just checked that out). No issue required.

Reply:
done